### PR TITLE
feat: fix html style and rollback old upload component

### PR DESCRIPTION
**Pull request by o0OooO**
## Description

Please include a summary of the changes and the related issue. Please also include relevant motivation and context. List any dependencies that are required for this change.

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

### CODE REVIEW

File: aws_extension/sagemaker_ui.py [hunk 1]
```diff
@@ -494,7 +494,118 @@ def refresh_all_models():
         print(f"Error refresh all models: {e}")
 
 
-def sagemaker_upload_model_s3():
+def sagemaker_upload_model_s3(sd_checkpoints_path, textual_inversion_path, lora_path, hypernetwork_path, controlnet_model_path):
+    log = "start upload model to s3:"
+
+    local_paths = [sd_checkpoints_path, textual_inversion_path, lora_path, hypernetwork_path, controlnet_model_path]
+
+    print(f"Refresh checkpionts before upload to get rid of duplicate uploads...")
```
Comment (NingLu) [inline aws_extension/sagemaker_ui.py:502]: Typo - "checkpoints"

**Review by NingLu — APPROVED** [review]

File: aws_extension/sagemaker_ui.py (hunk 1 above)
Comment (yike5460) [inline aws_extension/sagemaker_ui.py:502]: typo, checkpoints

File: scripts/main.py [hunk 2]
```diff
@@ -828,7 +828,72 @@ def update_api_key():
...
+                        "control": os.path.join(root_path, "models", "ControlNet"),
+                        "hyper": os.path.join(root_path, "models", "hypernetworks"),
+                    }
+                    def scan_sd_ckpt():
+                        model_files = os.listdir(model_folders["ckpt"])
+                        # filter non-model files not in exts
+                        model_files = [f for f in model_files if os.path.splitext(f)[1] in exts]
+                        model_files = [os.path.join(model_folders["ckpt"], f) for f in model_files]
+                        return model_files
+                    def scan_textural_inversion_model():
```
Comment (yike5460) [inline scripts/main.py:848]: typo, textual

File: aws_extension/sagemaker_ui.py [hunk 3]
```diff
@@ -494,7 +494,118 @@ def refresh_all_models():
...
+    refresh_all_models()
+
+    for lp, rp in zip(local_paths, checkpoint_type):
+        if lp == "" or not lp:
+            continue
+        print(f"lp is {lp}")
```
Comment (yike5460) [inline aws_extension/sagemaker_ui.py:508]: use log instead

File: aws_extension/sagemaker_ui.py [hunk 4]
```diff
@@ -494,7 +494,118 @@ def refresh_all_models():
...
+            # Upload src model to S3.
+            if rp != "embeddings" :
+                local_model_path_in_repo = os.sep.join(['models', rp, model_name])
+            else:
+                local_model_path_in_repo = os.sep.join([rp, model_name])
+            #local_tar_path = f'{model_name}.tar'
+            print("Pack the model file.")
+            # os.system(f"cp -f {lp} {local_model_path_in_repo}")
+            cp(lp, local_model_path_in_repo, recursive=True)
+            if rp == "Stable-diffusion":
```
Comment (yike5460) [inline aws_extension/sagemaker_ui.py:564]: what are the other type else except "stable-diffusion", "embedding"?

**Review by yike5460 — CHANGES_REQUESTED** [review]
some typos along with comments inside

File: scripts/main.py [hunk 5]
```diff
@@ -828,7 +828,72 @@ def update_api_key():
...
+                    gr.HTML(value="Refresh to select the model to upload to S3")
+                    exts = (".bin", ".pt", ".pth", ".safetensors", ".ckpt")
+                    root_path = os.getcwd()
+                    model_folders = {
+                        "ckpt": os.path.join(root_path, "models", "Stable-diffusion"),
+                        "text": os.path.join(root_path, "embeddings"),
+                        "lora": os.path.join(root_path, "models", "Lora"),
+                        "control": os.path.join(root_path, "models", "ControlNet"),
+                        "hyper": os.path.join(root_path, "models", "hypernetworks"),
+                    }
```
Comment (alvindaiyan) [inline scripts/main.py:841]: we need add vae type?

File: javascript/txt2img-config.js [hunk 6]
```diff
@@ -93,12 +93,13 @@ function showFileName(event) {
             const fileName = uploadedFile.name;
             const fileSize = uploadedFile.size;
             const fileType = uploadedFile.type;
-            const fileItemDiv = document.createElement("div");
+            const fileItemDiv = document.createElement("tr");
```
Comment (alvindaiyan) [inline javascript/txt2img-config.js:96]: `<tr>` should be in `<table>` section, and column need in `<th>` section

File: scripts/main.py [hunk 7]
```diff
@@ -828,7 +828,72 @@ def update_api_key():
...
+                        model_files = [os.path.join(model_folders["control"], f) for f in model_files]
+                        return model_files
+                    def scan_hypernetwork_model():
+                        model_files = os.listdir(model_folders["hyper"])
+                        # filter non-model files not in exts
+                        model_files = [f for f in model_files if os.path.splitext(f)[1] in exts]
+                        model_files = [os.path.join(model_folders["hyper"], f) for f in model_files]
+                        return model_files
+
+                    with FormRow(elem_id="model_upload_form_row_01"):
```
Comment (alvindaiyan) [inline scripts/main.py:873]: element id is duplicated to Line 898

File: aws_extension/sagemaker_ui.py (hunk 4 above)
Comment (o0OooO) [reply aws_extension/sagemaker_ui.py:565]: ["Stable-diffusion", "embeddings", "Lora", "hypernetworks", "ControlNet"]

Comment (o0OooO) [reply aws_extension/sagemaker_ui.py:565]: 这个原有的上传逻辑没变，VAE的上传放在新的upload组件中了

**Review by alvindaiyan — CHANGES_REQUESTED** [review]

File: scripts/main.py (hunk 5 above)
Comment (o0OooO) [reply scripts/main.py:841]: 在新上传组件里 这个原有的保持不变

File: javascript/txt2img-config.js (hunk 6 above)
Comment (o0OooO) [reply javascript/txt2img-config.js:96]: 只用来对齐

**Pull request merged**